pkg/stable.packages.props

```diff
@@ -2,16 +2,16 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <ItemGroup>
     <!-- NETCore 1.0 core-setup -->
-    <StablePackage Include="Microsoft.NETCore.App">
+    <!--<StablePackage Include="Microsoft.NETCore.App">
       <Version>1.0.0</Version>
-    </StablePackage>
+    </StablePackage>-->
     <StablePackage Include="Microsoft.NETCore">
```

Comment:
Do these still need to be here?  Why are these packages in twice?  Since they use the same Include, doesn't the bottom ones just overwrite these?